contrib/benchmark.py

```diff
@@ -160,7 +160,10 @@ def do_write(dst):
     out_speed = dict()
     for alg in ALGS:
         log.info('compressing with %s-6...', alg)
-        backend = ComprencBackend(b'pass', (alg, 6), Backend('local://' + backend_dir, None, None))
+        temp_parser = argparse.ArgumentParser()
+        temp_parser.add_argument('--storage_url')
+        temp_options = temp_parser.parse_args(['--storage_url','local://' + backend_dir])
```

Comment:
Thanks! This solution is somewhat verbose though. Could you just use a Namespace object like the test cases do? See https://github.com/s3ql/s3ql/blob/master/tests/t1_backends.py#L184 for an example.